osu.Game.Tournament/Screens/Editors/Components/LadderResetTeamsDialog.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Overlays.Dialog;
+using osu.Game.Tournament.Screens.Ladder.Components;
+
+namespace osu.Game.Tournament.Screens.Editors.Components
+{
+    public partial class LadderResetTeamsDialog : DangerousActionDialog
+    {
+        public LadderResetTeamsDialog(Container<DrawableTournamentMatch> matchesContainer)
+        {
+            HeaderText = @"Confirm reset teams?";
```

Comment:
```suggestion
            HeaderText = @"Reset teams?";
```

I can't say I like this phrasing with "confirm" jammed in there at the start. Reads wrong.